packages/core/js/index.d.ts

```diff
@@ -33,47 +33,47 @@ export class AEPCore{
     static setAppGroup(appGroup?: string);
     static resetIdentities();
 }
-export class AEPLifecycle{
+export class Lifecycle{
     static extensionVersion(): Promise<string>;
 }
-export class AEPIdentity{
+export class Identity{
     static extensionVersion(): Promise<string>;
     static syncIdentifiers(identifiers?: Map);
     static syncIdentifiersWithAuthState(identifiers?: Map, authenticationState: string);
     static syncIdentifier(identifierType: string, identifier: string, authenticationState: string);
     static appendVisitorInfoForURL(baseURL?: string): Promise<?string>;
     static getUrlVariables(): Promise<?string>;
-    static getIdentifiers(): Promise<Array<?AEPVisitorID>>;
+    static getIdentifiers(): Promise<Array<?VisitorID>>;
     static getExperienceCloudId(): Promise<?string>;
 }
-export class AEPSignal{
+export class Signal{
     static extensionVersion(): Promise<string>;
 }
 
-export class AEPMobilePrivacyStatus{
+export class PrivacyStatus{
     static OPT_IN: string;
     static OPT_OUT: string;
     static UNKNOWN: string;
 }
-export class AEPMobileLogLevel{
+export class LogLevel{
     static ERROR: string;
     static WARNING: string;
     static DEBUG: string;
     static VERBOSE: string;
 }
-export class AEPMobileVisitorAuthenticationState{
+export class MobileVisitorAuthenticationState{
     static AUTHENTICATED: string;
     static LOGGED_OUT: string;
     static UNKNOWN: string;
 }
-export class AEPVisitorID{
+export class VisitorID{
```

Comment:
Should we rename this to Identifiable/CustomIdentity as swift AEPCore? I feel like this can be confused with the VID. - this applies for the JS file too

Reply:
```java
public static void getIdentifiers(final AdobeCallback<List<VisitorID>> callback);
```

```swift
static func getIdentifiers(completion: @escaping ([Identifiable]?, Error?) -> Void)
```

I think `Identifiable` is too _**Swifty**_, so I have to choose the class name we exposed in Android SDKs. IMO we'd better not pick a new class name for JS APIs.